integration/isolated/delete_buildpack_command_test.go

```diff
@@ -0,0 +1,46 @@
+package isolated
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	. "code.cloudfoundry.org/cli/integration/helpers"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+	. "github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("delete-buildpack command", func() {
+	Context("when the stack is specified to disambiguate multiple buildpacks with the same name", func() {
+		var dir string
+
+		BeforeEach(func() {
+			LoginCF()
+
+			var err error
+			dir, err = ioutil.TempDir("", "delete-buildpack-test")
+			Expect(err).ToNot(HaveOccurred())
+
+			filename := "manifest.yml"
+			manifestFile := filepath.Join(dir, filename)
+			err = ioutil.WriteFile(manifestFile, []byte("---\nstack:cflinuxfs2"), 0400)
+			Expect(err).ToNot(HaveOccurred())
+
+			session := CF("create-buildpack", "some-buildpack", dir, "1")
```

Comment:
Looks like only one buildpack is being created here; this test should not depend on the behavior of `update-buildpack` integration test creating a buildpack with the same name.

Reply:
👍